services/core-web/common/components/PartyAppointmentTable.js

```diff
@@ -180,9 +180,7 @@ const PartyAppointmentTable = (props) => {
   return (
     <Row>
       <Col span={24}>
-        <Typography.Title level={3} className="margin-large--top">
-          Historical Engineer of Record List
-        </Typography.Title>
+        <h3>Historical Engineer of Record List</h3>
```

Comment:
Any particular reason for changing to h3 ?

Reply:
@henryoforeh-aot 
To keep the consistency I was following Project summary page (discussed with Denise).
![image](https://github.com/bcgov/mds/assets/118843449/b2087ce5-5e80-4360-9abe-ea3c78efc20b)

And that corresponding code used the `<h3>` as below.
https://github.com/bcgov/mds/blob/develop/services/core-web/src/components/Forms/projectSummaries/ProjectSummaryForm.js#L449

 I tried `<Typography.Title level={3}..` and leve={4}` But it doesn't provide the expected heading sizes. Any other better way to get this done?

Reply:
@henryoforeh-aot 
As discussed, updated all the `h3` to  `<Typography.Title level={4}..`.